packages/material-ui/src/TableBody/TableBody.js

```diff
@@ -10,28 +11,22 @@ export const styles = {
   },
 };
 
-class TableBody extends React.Component {
-  getChildContext() {
-    // eslint-disable-line class-methods-use-this
-    return {
-      tablelvl2: {
-        variant: 'body',
-      },
-    };
-  }
+function TableBody(props) {
+  const { classes, className, component: Component, ...other } = props;
+  const childContext = { variant: 'body' };
```

Comment:
As an easy perf optimization I would suggest moving this to a string only. Otherwise a rerender of TableBody will trigger a rerender of every context consumer since strict equality is used for rerender check.